Makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	@gox -osarch "darwin/amd64 linux/amd64" -output "dist/{{.Dir}}_{{.OS}}_{{.Arch}}"
+	@gox -osarch "darwin/amd64 linux/amd64" -output "dist/{{.Dir}}_{{.OS}}_{{.Arch}}" helm-update-config
 
 release:
 	@ghr -t ${GITHUB_TOKEN} -u burdiyan -r helm-update-config --replace `git describe --tags` dist/
```

Comment:
Hi @EmilyEmily ,

Would you please help update the release and the readme doc, including use name(burdiyan), install guide and usage, as we'll maintain our own repo in future. Thanks!

Reply:
Thanks for the reminder, updated releases and readme doc. 
I'm working on ginkgo unit test now, and will push a new version later.

Reply:
done